<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContainerPage</class>
 <widget class="QWidget" name="ContainerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>491</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">#leftPane, #rightPane {
background-color: #ffffff;
}
#leftPane {
border: solid #F3F5F7;
border-width: 0px 1px 0px 0px;
}
#containerHeader {
background-color: #F3F5F7;
}
#navigationArea {
background-color: #F3F5F7;
border-top: 1px solid #E7EAEF;
}</string>
  </property>
  <layout class="QVBoxLayout" name="ContainerPageLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="containerHeader" native="true">
     <property name="minimumSize">
      <size>
       <width>914</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
font-family: Roboto, Helvetica;
font-size: 14px;
}
#container {
color: #415982;
}
#containerFile {
color: #215081;
text-decoration: underline solid;
}
QPushButton {
padding: 9px 12px;
border-radius: 4px;
color: #2F70B6;
font-weight: 700;
}
QPushButton:hover {
background-color: #EAF1F8;
}
QPushButton:pressed {
background-color: #BFD3E8;
}</string>
     </property>
     <layout class="QHBoxLayout" name="containerHeaderLayout" stretch="0,1,0">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="container">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>Container:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="containerFile">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string notr="true">C:\Downloads\Leping.bdoc</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changeLocation">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Change</string>
        </property>
        <property name="icon">
         <iconset resource="../images/images.qrc">
          <normaloff>:/images/icon_Edit.svg</normaloff>:/images/icon_Edit.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="containerListsLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="FileList" name="leftPane"/>
     </item>
     <item>
      <widget class="ItemList" name="rightPane"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="navigationArea" native="true">
     <property name="minimumSize">
      <size>
       <width>914</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
padding: 9px 12px;
border-radius: 4px;
color: #2F70B6;
font-family: Roboto, Helvetica;
font-size: 12px;
font-weight: 700;
}
QPushButton:hover {
background-color: #EAF1F8;
}
QPushButton:pressed {
background-color: #BFD3E8;
}
QPushButton:default {
color: #AD2A45;
}
QPushButton:default:hover {
background-color: #F5EBED;
}
QPushButton:default:pressed {
background-color: #E1C1C6;
}</string>
     </property>
     <layout class="QHBoxLayout" name="navigationAreaLayout">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>32</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="../images/images.qrc">
          <normaloff>:/images/icon_arrow_left.svg</normaloff>:/images/icon_arrow_left.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="actionHiddenSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="convert">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Encrypt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveAs">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Save as</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="email">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Send with e-mail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="summary">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Print summary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Save without signing</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="mainActionSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemList</class>
   <extends>QScrollArea</extends>
   <header>widgets/ItemList.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileList</class>
   <extends>QScrollArea</extends>
   <header>widgets/FileList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
